models/opensea/opensea_events.sql

```diff
@@ -55,36 +55,36 @@ FROM
                 project,
                 version,
                 block_time,
-                NULL::string as token_id,
-                NULL::string as collection,
+                CAST(NULL AS VARCHAR(5)) as token_id,
+                CAST(NULL AS VARCHAR(5)) as collection,
                 amount_usd,
                 token_standard,
-                NULL::string as trade_type,
-                NULL::string as number_of_items,
-                NULL::string as trade_category,
+                CAST(NULL AS VARCHAR(5)) as trade_type,
+                CAST(NULL AS VARCHAR(5)) as number_of_items,
+                CAST(NULL AS VARCHAR(5)) as trade_category,
                 evt_type,
-                NULL::string as seller,
-                NULL::string as buyer,
+                CAST(NULL AS VARCHAR(5)) as seller,
+                CAST(NULL AS VARCHAR(5)) as buyer,
                 amount_original,
                 amount_raw,
                 currency_symbol,
                 currency_contract,
-                NULL::string as nft_contract_address,
+                CAST(NULL AS VARCHAR(5)) as nft_contract_address,
                 project_contract_address,
-                NULL::string as aggregator_name,
-                NULL::string as aggregator_address,
+                CAST(NULL AS VARCHAR(5)) as aggregator_name,
+                CAST(NULL AS VARCHAR(5)) as aggregator_address,
                 tx_hash,
                 block_number,
-                NULL::string as tx_from,
-                NULL::string as tx_to,
+                CAST(NULL AS VARCHAR(5)) as tx_from,
+                CAST(NULL AS VARCHAR(5)) as tx_to,
                 NULL::double as platform_fee_amount_raw,
                 NULL::double as platform_fee_amount,
                 NULL::double as platform_fee_amount_usd,
-                NULL::string as platform_fee_percentage,
+                CAST(NULL AS VARCHAR(5)) as platform_fee_percentage,
                 NULL::double as royalty_fee_amount_raw,
                 NULL::double as royalty_fee_amount,
                 NULL::double as royalty_fee_amount_usd,
```

Comment:
Wouldn't the `::double` fail on TrinoSQL as well?

Reply:
Yes, you're right, let me fix those